packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -43,15 +44,35 @@ class DedupeStorage extends BaseMockBlobStorage {
 	public async createBlob(blob: ArrayBufferLike) {
 		const id = await gitHashFile(blob as any);
 		this.blobs.set(id, blob);
-		return { id };
+		return { id, minTTLInSeconds: MIN_TTL };
+	}
+}
+
+class DedupeTTLStorage extends BaseMockBlobStorage {
```

Comment:
I don't think a new class is necessary. It should be fine to directly manipulate the TTL as needed since these are mocks.